### PR TITLE
Add verify and merge jobs for various devices

### PR DESCRIPTION
**Pull request by JPWKU**
Signed-off-by: Jeremy Phelps <jphelps@linuxfoundation.org>

### CODE REVIEW

File: jjb/device/device-bacnet.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+
+- project:
+    name: device-backnet
```
Comment (JPWKU) [inline jjb/device/device-bacnet.yaml]: I just noticed I spelled `backnet` instead of `bacnet`

**Comment by JPWKU** [comment]
@trcox @jpwhitemn @tykeal Just setting up the Nexus bits and maven settings file for these projects.

**Comment by JPWKU** [comment]
@tykeal @zxiiro I've added the settings and Nexus bits required.  This is ready for review.

**Review by tykeal — DISMISSED** [review]

**Comment by tykeal** [comment]
@JPWKU could you please rebase this so we can verify against the updated global-jjb v0.7.0

**Review by tykeal — APPROVED** [review]

**Pull request merged**